### ISSUE TITLE
Release of ego.powerflow

### ISSUE THREAD

**Issue by gplssm** [openego/ego.powerflow#29]
The RLI wants to release a version of Dingo. This is dependent on ego.io and ego.powerflow. In order to release a straightforward installable version, we would like to release ego.powerflow in its current state. This means:
1. Dingo requires the structure of the branch _refactor/adaptions-of-tablenames_
2. It would make sense to include switch to PyPSA 0.8.0 as well

Therefore, I suggest that I first check if the branch _refactor/upgrade-to-pypsa-0.8_ if it runs properly with Dingo. If yes (what I assume), I merge this to dev. As far as I understand, this branch should include all changes of _refactor/adaptions-of-tablenames_.

@openego/ego-powerflow If you have objections, please tell!

Update: add more mentioning to reach all of you @lukasol @ulfmueller @IlkaCu @s3pp @nesnoj @wolfbunke @BartelsJ

**Comment by gplssm** [comment]
I have to postpone the release: found deviating results with different PyPSA versions and haven't found the reason. Once #30 is done, release could be done.

**Comment by gplssm** [comment]
BÄM https://pypi.python.org/pypi/egopowerflow/0.0.3 :tada:
